File: CS/10_Conversion/ConvertPdfWithDefaultFont.cs
using Spire.Presentation;
using Spire.Presentation.Charts;
using Spire.Presentation.Drawing;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace ConvertPdfWithDefaultFont
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void btnRun_Click(object sender, EventArgs e)
        {
            //Load PPT from disk
            Presentation ppt = new Presentation();
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\ConvertPdfWithDefaultFont.pptx");
            
            //The font is preferred to convert to pdf or pictures, when the font used in the document is not installed in the system
            Presentation.SetDefaultFontName("Arial");
            
            //Save to file
            ppt.SaveToFile("ConvertPdfWithDefaultFont_out.pdf", FileFormat.PDF);

            //Launch and view the resulted file
            PresentationDocViewer("ConvertPdfWithDefaultFont_out.pdf");
        }
        private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/10_Conversion/ConvertPdfWithDefaultFont.vb
Imports Spire.Presentation
Imports Spire.Presentation.Charts
Imports Spire.Presentation.Drawing
Imports System.ComponentModel
Imports System.Text

Namespace ConvertPdfWithDefaultFont
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Load PPT from disk
			Dim ppt As New Presentation()
			ppt.LoadFromFile("..\..\..\..\..\..\Data\ConvertPdfWithDefaultFont.pptx")

			'The font is preferred to convert to pdf or pictures, when the font used in the document is not installed in the system
			Presentation.SetDefaultFontName("Arial")

			'Save to file
			ppt.SaveToFile("ConvertPdfWithDefaultFont_out.pdf", FileFormat.PDF)

			'Launch and view the resulted file
			PresentationDocViewer("ConvertPdfWithDefaultFont_out.pdf")
		End Sub
		Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
